README.md

```diff
@@ -203,14 +205,14 @@
       selectedLocale: string;
       user: string;
       userId: string;
-      userAccountFeatureFlagsEnabled: boolean;
-      userAccountFeatureFlagsUserId: string;
-      initialContext?: [
+      initialContext: [
         {
           name: string;
+          type: string;
           value: any;
         }
-      ];
+      ] | null;
```

Comment:
this should be always there but then `[]` we have to make sure that property should be always there

Reply:
Updated to be always array